specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/guestDiagnosticSettings_API.json

```diff
@@ -0,0 +1,523 @@
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "diagnosticSettings",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsResource"
+                        },
+                        "description": "The configuration to create or update."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An existing guest diagnostic settings was successfully updated.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsResource"
+                        }
+                    },
+                    "201": {
+                        "description": "A new guest diagnostic settings was successfully created.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the guest diagnostic settings could not be created or updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },        
+            "get": {
+                "description": "Gets guest diagnostics settings.",
+                "tags": [
+                    "GuestDiagnosticsSettings"
+                 ],
+                "operationId": "guestDiagnosticsSettings_Get",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsResource"
+                        }
+                    },
+                    "404": {
```

Comment:
Shouldnt this be handled by the "default" error response? I dont think you need to add it explicitly

Reply:
Done